src/core/runtime/src/configuration/load-organizations-config.ts

```diff
@@ -388,13 +388,25 @@ async function validateScpsCount(
       Filter: 'SERVICE_CONTROL_POLICY',
       TargetId: ouObject.Id!,
     });
-    const accelScps = ouConfig.scps.map(policyName =>
+    const accelOuScps = ouConfig.scps.map(policyName =>
       ServiceControlPolicy.policyNameToAcceleratorPolicyName({ acceleratorPrefix, policyName }),
     );
+    const configScps = config['global-options'].scps;
+    const accelScps = configScps.map(scp =>
+      ServiceControlPolicy.policyNameToAcceleratorPolicyName({
+        acceleratorPrefix,
+        policyName: scp.name,
+      }),
+    );
     const nonAccelScps = attachedScps.filter(as => !accelScps.includes(as.Name!));
-    if (nonAccelScps.length + accelScps.length > MAX_SCPS_ALLOWED) {
+    if (accelOuScps.length > MAX_SCPS_ALLOWED) {
```

Comment:
This new error will be confusing for customers - they define 6 scps on an ou, it errors telling them max is 5.  they remove 1 and rerun and then it will hit the very next check telling them they have still exceeded the limit because of existing SCPs on the ou.  I don't see this new error as being needed.